apps/services/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.template.defaultfilters import slugify
+from django.contrib.auth.models import User
+
+class Services(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
@KicaRonaldOkello , I think services shouldnot be tied to a stylist.
Services should be open for all stylists to add to there arsenal.

Reply:
I think services should be like categories